### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by AshakaE**
### In this milestone, I did
- Create repo with module-specific requirements
- Remove unwanted files and assets
- Add big.js
- Add prop-types 
- Deploy to heroku

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,17 +0,0 @@
...
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
-);
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
```
Comment (Torres-ssf) [inline src/index.js]: - [x] Here we can add the React render setup code

```JS
import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';

ReactDOM.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>,
  document.getElementById('root')
);
```

File: src/App.js [hunk 2]
```diff
@@ -1,25 +0,0 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
```
Comment (Torres-ssf) [inline src/App.js]: - [x] And then, here we can simply make `App` component to return a message:
```JS
function App() {
  return (
    <h1>Hello World</h1>
  );
}

export default App;
```

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes Required :muscle: 
Hello @AshakaE :hand: 
Awesome job on this milestone so far.
But we need to make some changes before moving on.

- [x] We need to push your app online and make the link available on the README file.
Screenshot from 2021-05-12 09-25-30

- [x] Awesome job delete all unnecessary assets. However, it seems you deleted more than it's required. We didn't need to delete the index.js file with code that makes your app connects with React. Check my comments below

Happy coding :computer: 
**Sérgio Torres**

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @AshakaE :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**